scanner/src/main/kotlin/ScannerCriteria.kt

```diff
@@ -138,8 +121,9 @@ data class ScannerCriteria(
             val minVersion = parseVersion(options[PROP_CRITERIA_MIN_VERSION]) ?: scannerVersion
             val maxVersion = parseVersion(options[PROP_CRITERIA_MAX_VERSION]) ?: minVersion.nextMinor()
             val name = options[PROP_CRITERIA_NAME] ?: details.name
+            val configuration = options[PROP_CRITERIA_CONFIGURATION] ?: details.configuration
```

Comment:
@mnonnenmacher, I just realized a general problem with this: `ScannerCriteria.configuration` is nullable to allow matching all configurations, but we can never set it to null via a scanner's `Options` as this will then fall back to `ScannerDetails.configuration` which is non-nullable.